README.md

```diff
@@ -54,7 +54,7 @@ section in the Zebra book for system requirements.
 This command will run our latest release, and sync it to the tip:
 
 ```sh
-docker run zfnd/zebra:1.0.0-rc.9
+docker run zfnd/zebra:1.0.0
```

Comment:
Now we are doing stable releases, we will automatically have a "latest" Docker tag that will get updated to the latest stable version
```suggestion
docker run zfnd/zebra:latest
```